ioctl/cmd/did/didgenerate.go

```diff
@@ -102,6 +103,8 @@ func generateFromSigner(signer, password string) (generatedMessage string, err e
 	if err != nil {
 		return "", output.NewError(output.ConvertError, "", err)
 	}
-	generatedMessage = string(msg)
+
+	sum := sha256.Sum256(msg)
+	generatedMessage = string(msg) + "\n\nThe sha256 hash is:" + hex.EncodeToString(sum[:])
```

Comment:
`The hex encoded SHA256 hash of the DID doc is:`

Reply:
done